rust/processor/src/db/common/models/ans_models/parquet_ans_lookup_v2.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+type Domain = String;
+type Subdomain = String;
+type TokenStandardType = String;
+type RegisteredAddress = String;
+// PK of current_ans_lookup_v2
+type CurrentAnsLookupV2PK = (Domain, Subdomain, TokenStandardType);
+// PK of current_ans_primary_name
+type CurrentAnsPrimaryNameV2PK = (RegisteredAddress, TokenStandardType);
+
+#[derive(Clone, Default, Debug, Deserialize, FieldCount, Serialize)]
+pub struct AnsPrimaryNameV2 {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub registered_address: String,
+    pub token_standard: String,
+    pub domain: Option<String>,
+    pub subdomain: Option<String>,
+    pub token_name: Option<String>,
+    pub is_deleted: bool,
+    pub block_timestamp: chrono::NaiveDateTime,
```

Comment:
should we have a `is_v1` flag?

Reply:
I think `token_standard = v1` means its for v1 
![Screenshot 2024-07-02 at 5 11 31 PM](https://github.com/aptos-labs/aptos-indexer-processors/assets/38443641/6616604b-19c1-4b61-8d93-92222e5662f2)